app/packs/src/components/common/ToggleButton.js

```diff
@@ -24,9 +28,9 @@ export default function ToggleButton({
         className={`toggle-button ${isToggled ? 'on' : 'off'}`}
         size="xs"
         onClick={handleChange}
-        style={{ backgroundColor: buttonColor, minWidth: '50px', border: 'none' }}
+        style={{ backgroundColor: buttonColor, border: 'none', padding: '0px 0px 0px 0px' }}
```

Comment:
This inline style can be replaced with corresponding bootstrap utility classes

Reply:
done